alf/bin/train_play_test.py

```diff
@@ -141,7 +141,10 @@ def _to_conf_params(parameters):
     'TrainerConfig.num_updates_per_train_iter=1',
     'TrainerConfig.mini_batch_length=2',
     'TrainerConfig.mini_batch_size=4',
-    'TrainerConfig.replay_buffer_length=64',
+    # If replay_buffer_length is above 6 (3 iters * 2 unroll length), whole replay buffer training
+    # algorithms (e.g. mbrl_pendulum) will not get enough training data, and will not start training.
+    # This fails during playing, when the config file isn't generated in root_dir.
```

Comment:
Why a longer replay buffer won't give enough training data?

Reply:
> because the replay buffer is cleared before it's fully filled. So training never starts.

Whole replay buffer training doesn't require the buffer to be full? It just gets all of data in the buffer even though then it's not full.

Just curious: this number of 64 had no issue before? Did you meet this problem for new test cases?